tests/contenttypes_tests/test_management.py

```diff
@@ -11,6 +12,17 @@
 
 
 class UpdateContentTypesTests(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.available_apps = [
+            'contenttypes_tests',
+            'contenttypes_tests.app_with_no_models',
+            'django.contrib.contenttypes',
+        ]
+        apps.set_installed_apps(settings.INSTALLED_APPS + ['contenttypes_tests.app_with_no_models'])
```

Comment:
Could not really avoid putting those lines, though I moved them to setUpClass from the class body.
The command that deletes the content types iterates over available apps. that's why I need the first line. "available_apps" needs to be a subset of "installed_apps", therefore a second line.

Is there a better way to do it?